MapboxDirections/MBRouteStep.swift

```diff
@@ -179,27 +179,41 @@ public enum ManeuverType: Int, CustomStringConvertible {
     case useLane
      
     /**
-     The step requires the user to enter, traverse, and exit a roundabout (traffic circle or rotary).
+     The step requires the user to enter and traverse a roundabout (traffic circle or rotary).
      
      The step has no name, but the exit name is the name of the road to take to exit the roundabout. The exit index indicates the number of roundabout exits up to and including the exit to take.
      */
     case takeRoundabout
     
     /**
-     The step requires the user to enter, traverse, and exit a large, named roundabout (traffic circle or rotary).
+     The step requires the user to enter and traverse a large, named roundabout (traffic circle or rotary).
      
      The step’s name is the name of the roundabout. The exit name is the name of the road to take to exit the roundabout. The exit index indicates the number of rotary exits up to and including the exit that the user must take.
      */
     case takeRotary
     
     /**
-     The step requires the user to enter and exit a roundabout (traffic circle or rotary) that is compact enough to constitute a single intersection.
+     The step requires the user to enter and  a roundabout (traffic circle or rotary) that is compact enough to constitute a single intersection.
```

Comment:
Stray “and”.

Reply:
Actually, bring back “exit”, because there’s no `exit roundabout turn` maneuver type.